frontend/src/components/pages/UserDashboard/Presentation.jsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Grid, Paper } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
-import TeamsTable from "../UserDetails/TeamsTable";
+import TeamsTable from "../../regions/UserDetails/TeamsTable";
```

Comment:
Since the TeamsTable is only used in this one place it would be best to move it. This will make the code base more cohesive and easier to navigate.

So the terams table compeonent files should be inside the UserDashboard component

Reply:
Done. Please check it out and let me know if the folder structure is agreeable